pkg/subctl/cmd/logs.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+)
+
+func getConnectivityPodsLogs(clientSet kubernetes.Interface) (bool, error){
```

Comment:
I don't see the purpose of returning a `bool` along with `error`. It seems the `bool` is to indicate the logs were successfully collected but a nil `error` also indicates that. So I'd just return `error`.